lintly/parsers.py

```diff
@@ -64,13 +64,22 @@ def parse_violations(self, output):
             if not match:
                 continue
 
-            path = self._normalize_path(match.group('path'))
+            def get_from_match(key, not_found_value=None):
+                if key in match.groupdict():
+                    return match.group(key)
+                return not_found_value
+
+            path = self._normalize_path(get_from_match("path"))
+            line = int(get_from_match("line", 0))
```

Comment:
My first reaction upon reading this was 'what's to distinguish between "we found an error which happened to be on line 0" vs. "we didn't find anything, so defaulting to line 0"?'.

Maybe add a comment here to remind future-me, something like 'linter does not report this field; fall back to reporting at the beginning of file' or something like that?